docs/_guide/decorators.md

```diff
@@ -0,0 +1,260 @@
+---
+layout: guide
+title: Using decorators
+slug: decorators
+---
+
+Decorators are special declarations that you can use when defining classes, class methods, and class fields. LitElement supplies a set of decorators that reduce the amount of boilerplate code you need to write when defining a component. 
+
+For example, the `customElement` and `property` decorators make a basic element definition more compact:
+
+```js
+import {LitElement, html, customElement, property} from 'lit-element';
+
+@customElement('my-element')
+class MyElement extends LitElement {
+
+ // Declare observed properties
+ @property()
+ adjective = 'awesome';
+
+ // Define the element's template
+ render() {
+   return html`<p>your ${adjective} template here</p>`;
+ }
+}
+```
+
+## Enabling decorators
+
+To use decorators, you need to use a compiler such as Babel or the TypeScript compiler.
+
+<div class="alert alert-info">
+
+**The decorators proposal**. Decorators are a stage 2 proposal for addition to the ECMAScript standard, which means they're not implemented in browsers yet. Compilers like Babel provide support for proposed features like decorators by compiling them into JavaScript a browser can run.
+
+</div>
+
+### To use decorators with TypeScript
+
+To use decorators with TypeScript, enable the `experimentalDecorators` compiler option.
+
+```json
+"experimentalDecorators": true,
+```
+
+Enabling `emitDecoratorMetadata` is not required and not recommended.
+
+### To use decorators with JavaScript
+
+If you're  JavaScript with Babel, you can enable decorators by adding  the following plugins:
+
+*   [`@babel/plugin-proposal-decorators`](https://babeljs.io/docs/en/babel-plugin-proposal-decorators). 
+*   [`@babel/plugin-proposal-class-properties`](https://babeljs.io/docs/en/babel-plugin-proposal-class-properties)
+
+To enable the plugins, you'd add code like this to your Babel configuration:
+
+```js
+plugins = [
+  '@babel/plugin-proposal-class-properties',
+  ['@babel/proposal-decorators', {decoratorsBeforeExport: true}],
+];
+```
+
+## LitElement decorators
+
+LitElement provides the following decorators:
+
+*   <code>[customElement](https://lit-element.polymer-project.org/api/modules/_lit_element_.html#customelement)</code>. Define a custom element.
```

Comment:
These links are not working.

Reply:
Oops. Yeah, that is some bad convert-from-Docs markup there.